pkg/workload/bank/bank.go

```diff
@@ -186,13 +209,17 @@ func (b *bank) Ops(
 	db.SetMaxIdleConns(b.connFlags.Concurrency + 1)
 
 	// TODO(dan): Move the various queries in the backup/restore tests here.
-	updateStmt, err := db.Prepare(`
-		UPDATE bank
-		SET balance = CASE id WHEN $1 THEN balance-$3 WHEN $2 THEN balance+$3 END
-		WHERE id IN ($1, $2)
-	`)
-	if err != nil {
-		return workload.QueryLoad{}, errors.CombineErrors(err, db.Close())
+	updateStmts := make([]*gosql.Stmt, b.numTables)
+	for tableIdx := 0; tableIdx < b.numTables; tableIdx++ {
```

Comment:
nit: replace with for-range loop

```suggestion
	for tableIdx := range b.numTables {
```